Header header
float32 velocity # linear or angular, dependent on segment type
